export default (): Promise<Auditorio[]> => {
	return new Promise(async (resolve, reject) => {
		const auditorios = await Auditorio.find()
			.sort({ nome: 1 })
			.catch((err) => {
				new Erro({
					erro: {
						info: 'N찾o foi possivel ler os audit처rios do banco de dados',
						err,
					},
				}).save()
				return reject('Erro ao baixar audit처rios')
			})
		if (auditorios)
			return resolve(auditorios)
		return reject('Erro ao baixar audit처rios')
	})
}
